packages/ast-spec/src/base/BaseNode.ts

```diff
@@ -3,12 +3,8 @@ import type { AST_NODE_TYPES } from '../ast-node-types';
 import type { NodeOrTokenData } from './NodeOrTokenData';
 
 export interface BaseNode extends NodeOrTokenData {
-  /**
-   * The parent node of the current node
-   *
-   * This is added in the @typescript-eslint/types package as ESLint adds it
-   * while traversing.
-   */
+  // The parent node of the current node
+  // (added in @typescript-eslint/types as ESLint adds it while traversing)
```

Comment:
Switched to an inline `//` comment so the `/** */` jsdoc comment doesn't risk getting attached to `type`.